storage/command_queue_test.go

```diff
@@ -171,3 +171,19 @@ func TestCommandQueueClear(t *testing.T) {
 		t.Fatal("commands should finish when clearing queue")
 	}
 }
+
+// TestCommandQueueExclusiveEnd verifies that an end key is treated as
+// an exclusive end when GetWait calculates overlapping commands. Test
+// it by calling GetWait with a command whose start key is equal to
+// the end key of a previous command.
+func TestCommandQueueExclusiveEnd(t *testing.T) {
```

Comment:
Still confused. You're testing something about the command queue, but you haven't changed any of its code. You've only added something in the GC queue, but I don't see how your test exercises that new code path.